hardware/setup/snap_config.sh

```diff
@@ -64,3 +64,11 @@ echo -e "\t                        generating $NAME"
 if ([ "$NAME" == "snap_core_types.vhd" ]); then
   sed -i 's/CONSTANT NUM_OF_ACTIONS[ ^I]*:[ ^I]*integer.*:=[ ^I]*[0-9]*/CONSTANT NUM_OF_ACTIONS                  : integer := '$NUM_OF_ACTIONS'/' $2
 fi
+
+if [ "$NAME" == "psl_fpga.vhd" ]; then
+  if [ "${GOLDEN_IMAGE^^}" == "TRUE" ]; then
```

Comment:
The file name is case sensitive, it's always lower-case. 
GOLDEN_IMAGE could also be "TrUe", therefore ^^ makes it all uppercase before comparing